src/screens/reader/components/ReaderFooter.tsx

```diff
@@ -68,7 +71,13 @@ const ChapterFooter = ({
         <Pressable
           android_ripple={rippleConfig}
           style={styles.buttonStyles}
-          onPress={() => WebBrowser.openBrowserAsync(chapterUrl)}
+          onPress={() =>
+            navigate('WebviewScreen', {
+              name: `${chapter.name} | ${novel.name}`,
+              url: chapter.path,
+              pluginId: novel.pluginId,
+            })
+          }
         >
           <IconButton icon="earth" size={26} iconColor={theme.onSurface} />
         </Pressable>
```

Comment:
Could you add condition `novel.pluginId !== 2`. Because local plugin is not an actual instance

Reply:
I added hiding the icon with `islocal`.